src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -945,15 +944,33 @@ public static string XAsTooltip(string x)
             return $"X = {x}";
          }
 
-         public static string BoxWhiskerYAsTooltip(string lowerWhisker, string lowerBox, string median, string upperBox, string upperWhisker, string[] outliers)
+         public static string BoxWhiskerYAsTooltip(string lowerWhisker, int lowerWiskerIndividualId, string lowerBox, int lowerBoxIndividualId, string median, int medianIndividualId, string upperBox, int upperboxIndividualId, string upperWhisker,int  upperWhiskerIndividualId,  string[] outliers, int[] outlierIndividualIds)
          {
-            var outlierString = String.Empty;
-            if (outliers.Length > 0)
-               outlierString = $"\nOutliers: {string.Join(", ", outliers)}";
+            var sb = new StringBuilder();
 
-            return $"95% = {upperWhisker}\n75% = {upperBox}\n50% = {median}\n25% = {lowerBox}\n  5% = {lowerWhisker}{outlierString}";
+            sb.AppendLine(percentilWithIndividualId("95", upperWhisker, upperWhiskerIndividualId));
+            sb.AppendLine(percentilWithIndividualId("75", upperBox, upperboxIndividualId));
+            sb.AppendLine(percentilWithIndividualId("50", median, medianIndividualId));
+            sb.AppendLine(percentilWithIndividualId("25", lowerBox, lowerBoxIndividualId));
+            sb.AppendLine(percentilWithIndividualId("05", lowerWhisker, lowerWiskerIndividualId));
+
+            if (outliers.Length > 0 && outliers.Length == outlierIndividualIds.Length)
+            {
+               sb.AppendLine();
+               sb.AppendLine("<b>Outliers</b>");
+               outliers.Each((v, i) =>
+               {
+                  sb.AppendLine(valueWithIndividualId(v,outlierIndividualIds[i]));
+               });
+            }
+
+            return sb.ToString();
          }
 
+         private static string percentilWithIndividualId(string percentil, string value, int individualId) => valueWithIndividualId($"{percentil}% = {value}", individualId);
+
+         private static string valueWithIndividualId(string value, int individualId) => $"{value}, IndividualId = {individualId}";
```

Comment:
@Yuri05  This line may have to be changed depending on how we want to display the tooltip